ItemRack/ItemRackEvents.lua

```diff
@@ -291,15 +291,20 @@ function ItemRack.ProcessingFrameOnEvent(self,event,...)
 	local enabled = ItemRackUser.Events.Enabled
 	local events = ItemRackEvents
 	local startBuff, startZone, startStance, eventType
-	local arg1,arg2 = ...;
+	local arg1, arg2 = ...;
 
+	--  I'm try to prevent toggle set change when player change subzone but not in raid instance. it caused trinkets get 30s cooldown frequently.
+	local _, instanceType = IsInInstance()
```

Comment:
Please move this to line 307 with an extra if, because you only need it there, then you don't need to call IsInInstance() if you don't need it (like for aura events and so on)

Reply:
OK